docs/patterns/single-new-eks-observability-accelerators/single-new-eks-awsnative-fargate-observability.md

```diff
@@ -0,0 +1,162 @@
+# Single New EKS Fargate Cluster AWS Native Observability Accelerator
+
+## Architecture
+
+The following figure illustrates the architecture of the pattern we will be deploying for Single EKS Fargate Cluster Native Observability pattern using AWS native tools such as CloudWatch Logs and Container Insights.
+
+![Architecture](../images/cloud-native-arch.png)
+
+This example makes use of CloudWatch Container Insights as a vizualization and metric-aggregation layer.
+Amazon CloudWatch Container Insights helps customers collect, aggregate, and summarize metrics and logs from containerized applications and microservices. Metrics data is collected as performance log events using the embedded metric format. These performance log events use a structured JSON schema that enables high-cardinality data to be ingested and stored at scale. From this data, CloudWatch creates aggregated metrics at the cluster, node, pod, task, and service level as CloudWatch metrics. The metrics that Container Insights collects are available in CloudWatch automatic dashboards.
```

Comment:
Refer [this](https://aws-observability.github.io/observability-best-practices/guides/containers/aws-native/eks/amazon-cloudwatch-container-insights/#:~:text=Figure%3A%20Pipeline%20components%20in%20an%20ADOT%20Collector%20instance%20deployed%20to%20Amazon%20EKS%20with%20Fargate%20profile) BP guide on how Fargate CI Mon works and you can pick some verbiage from that. Above is not fully accurate.